### DISCUSSION TITLE
IEEE Quantum Week Tutorials

### DISCUSSION THREAD

**Discussion by gprs1809** [finos/zenith#93] [General]
The IEEE Quantum week will start from 17th September and will last till 22nd September. It is going to be a week with several interesting tutorial sessions on Quantum Computing basics, Quantum Error Mitigation, Quantum Cryptography, Quantum hardware among many other topics that can be found [here](https://qce.quantum.ieee.org/2023/tutorials-program/). Apart from the tutorial sessions, there will be many other events. Make sure to check the [site](https://qce.quantum.ieee.org/2023/). Though it appears that the events will be held at Washington, I have heard that there will be several virtual events as well.

**Comment by tenatzen** [comment]
Cheers for the heads up! Are they going to be doing any accreditations for people participants in the tutorials?

**Reply by gprs1809** [reply]
Thanks Keith! I don't think there will be any accreditations. They haven't mentioned anything like that in most of the tutorial descriptions that I saw. I can try finding out if there are accreditations.
